src/css/colors.css

```diff
@@ -2,6 +2,7 @@
 	--color-white: #ffffff;
 	--color-beige: #f7f6f2;
 
+	--color-gray-000: #979797;
```

Comment:
I noticed that this color has slightly more saturation that `--color-gray-200`. I think that this is okay as-is for this PR. For the sake of consistency with the broader design system, if it sounds reasonable, I could create a separate PR to do the following: 

- Do a find-and-replace for gray shades 300-600 that transfers them to 400-700
- In another commit, rename `--color-gray-000` to `--color-gray-300`